openmetadata-ui/src/main/resources/ui/src/components/GlobalSetting/GlobalSettingLeftPanel.tsx

```diff
@@ -16,21 +16,33 @@ import { ItemType } from 'antd/lib/menu/hooks/useItems';
 import { camelCase } from 'lodash';
 import React from 'react';
 import { useHistory, useParams } from 'react-router-dom';
+import { useAuthContext } from '../../authentication/auth-provider/AuthProvider';
 import { GLOBAL_SETTINGS_MENU } from '../../constants/globalSettings.constants';
+import { useAuth } from '../../hooks/authHooks';
 import { getGlobalSettingMenus } from '../../utils/GlobalSettingsUtils';
 import { getSettingPath } from '../../utils/RouterUtils';
 
 const GlobalSettingLeftPanel = () => {
   const { tab, settingCategory } = useParams<{ [key: string]: string }>();
+  const { isAdminUser } = useAuth();
+  const { isAuthDisabled } = useAuthContext();
 
   const history = useHistory();
-  const items: ItemType[] = GLOBAL_SETTINGS_MENU.map(({ category, items }) => {
+  const items: ItemType[] = GLOBAL_SETTINGS_MENU.filter(({ isProtected }) => {
+    if (isAdminUser || isAuthDisabled) {
+      return true;
+    }
+
+    return !isProtected;
+  }).map(({ category, items }) => {
     return getGlobalSettingMenus(
       category,
       camelCase(category),
       '',
+      '',
       items,
-      'group'
+      'group',
+      isAdminUser || isAuthDisabled
```

Comment:
Can we move this to yop level const and use it everywhere?

Reply:
about this, the 'group' is only needed for only this particular scenario, and we are not using it anywhere

Reply:
@ShaileshParmar11 , I was talking about this `  isAdminUser || isAuthDisabled` one.

Reply:
Thanks, from 1st comment it was not clear, thanks for clearing out!